replication/README.md

```diff
@@ -390,6 +390,29 @@ message GetVolumeReplicationInfoResponse {
   // This field is OPTIONAL.
   // The value of this field MUST NOT be negative.
   int64 last_sync_bytes = 3;
+  // Represents the type of replication status that can be returned
+  // by a storage vendor.
+  enum Status {
+    // UNKNOWN indicates that the CSI-driver does not support/know
+    // the current replication status.
+    UNKNOWN = 0;
+    // HEALTHY indicates that the replication is working properly.
+    HEALTHY = 1;
+    // DEGRADED indicates that the replication has stopped working
+    // and it may require user intervention.
+    DEGRADED = 2;
+    // ERROR indicates that there is an error in replication
+    // and it may require user intervention.
+    ERROR = 3;
+  }
+  // Represents the current replication status as reported by 
+  // the backend storage system.
+  // This field is OPTIONAL.
```

Comment:
Why is this OPTIONAL? Is the default not `UNKNOWN` so that it can be marked REQUIRED?

Reply:
How should the difference between `UNKNOWN` and an unset `.status` be handled? I suspect that both are the same for anyone that inspects the result of the procedure call.

If there is a good reason to differentiate between unset and `UNKNOWN`, I agree that OPTIONAL makes sense.

Reply:
It is still not clear to me how users should interpret the difference between the unavailable state, and a `.status=UNKNOWN`.

If users are expected to act differently on unset vs `UNKNOWN`, then the `.status` can be optional.

Otherwise if a status, either `UNKNOWN` or not-implemented, would be the same for the user, then make it required.

A status of `UNKNOWN` returned by the storage provider does not make a difference for the spec. A CSI-driver can select a more suitable status or elaborate the unknown-ness in the optional `.status_message`.

Reply:
Okay, updated it to to `REQUIRED`, PTAL.